$FreeWRT$
$MirOS: ports/misc/mc/patches/patch-src_main_c,v 1.6 2006/10/28 22:35:48 tg Exp $
--- src/main.c	2005-07-23 16:52:02.000000000 +0000
+++ src/main.c	2007-03-20 13:32:10.000000000 +0000
@@ -150,7 +150,7 @@ int auto_save_setup = 1;
 
 #ifndef HAVE_CHARSET
 /* If true, allow characters in the range 160-255 */
-int eight_bit_clean = 1;
+int eight_bit_clean = 0;
 
 /*
  * If true, also allow characters in the range 128-159.
